packages/i18n/src/index.js

```diff
@@ -69,15 +57,13 @@ function getI18n() {
  *
  * @return {string} The translated string.
  */
-const dcnpgettext = memoize( ( domain = 'default', context, single, plural, number ) => {
-	try {
-		return getI18n().dcnpgettext( domain, context, single, plural, number );
-	} catch ( error ) {
-		logErrorOnce( 'Jed localization error: \n\n' + error.toString() );
```

Comment:
The previous `try` / `catch` existed largely because Jed is rather `throw`-happy with how it deals with unexpected inputs. Though it's similarly fair that Tannin, called incorrectly, could similarly throw. It's maybe curious to think on how much it could make sense to validate inputs from the `@wordpress/i18n`.

I'm not so sure about providing the missing key callback, only because for the default English locale, we don't provide any initialization data, meaning we _rely_ on the missing key behavior intentionally as the default behavior.